js/background.js

```diff
@@ -26,13 +27,8 @@ function hack_referer_header(details) {
 
   if (details.url.indexOf(".xiami.com/") !== -1) {
     add_origin = false;
-    referer_value = "https://www.xiami.com";
```

Comment:
please only submit code related th this PR, otherwise it's hard to find out why code changed later because commit history will only show PR commit message.

Reply:
OK👌下次注意这个问题。
删除这个是因为虾米的端口不需要referer信息。当然保留也没问题，估计他们后端也没人去统计异常数据更新版本了。